jekyll/_cci2/pipelines.adoc

```diff
@@ -11,18 +11,17 @@ contentTags:
 :page-layout: classic-docs
 :page-liquid:
 
-
 CircleCI pipelines are the highest-level unit of work in your CI/CD process. Defined by a configuration file (typically `.circleci/config.yml`), pipelines orchestrate the execution of _workflows_ which run _jobs_ to build, test, and deploy your code.
 
 Pipelines can be triggered in the following ways:
 
-* Automatically: code changes, scheduled events, custom webhooks
+* Automatically: Git events, scheduled events, custom webhooks
 * Manually: CircleCI app or API
 
 Pipelines provide a flexible framework for creating conditional workflows based on xref:workflows#using-filters-in-your-workflows[filters], xref:workflows#workflows-configuration-examples[job requirements], and xref:selecting-a-workflow-to-run-using-pipeline-parameters#[pipeline parameters], allowing you to customize your CI/CD process to fit your project's needs.
 
 == Pipelines dashboard
-Your pipelines dashboard displays recently-triggered pipelines for your organization. Use the various filters to find the pipelines you need. Click through to view workflow maps and individual job steps. Access quick controls to rerun and cancel workflows
+Your pipelines dashboard displays recently-triggered pipelines for your organization. Use the various filters to find the pipelines you need. Click through to xref:jobs-steps#access-jobs-and-steps-in-the-circleci-app[view workflow maps and individual job steps]. Access quick controls to rerun and cancel workflows
```

Comment:
I think it's missing a full stop + I'm not actually sure what "access quick controls" means - I guess the 3 dots?